x-pack/plugins/ml/public/_app.scss

```diff
@@ -0,0 +1,30 @@
+// ML has app specific coloring for it's various warning levels. 
+// These are used almost everywhere.
+
+.ml-icon-severity-critical,
+.ml-icon-severity-major,
+.ml-icon-severity-minor,
+.ml-icon-severity-warning,
+.ml-icon-severity-unknown {
+  text-shadow: 1px 1px 1px $euiColorLightShade;
+}
+
+.ml-icon-severity-critical {
+  color: $mchColorCriticalText;
+}
+
+.ml-icon-severity-major {
+  color: $mchColorMajorText;
+}
+
+.ml-icon-severity-minor {
+  color: $mchColorMinorText;
+}
+
+.ml-icon-severity-warning {
+  color: $mchColorWarningText;
+}
+
+.ml-icon-severity-unknown {
+  color: $mchColorUnknownText;
+}
```

Comment:
Maybe we can apply the rule we use for JavaScript to have a newline as the last character of each file (see https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#newlines). We use these VS Code settings to do that automatically: https://github.com/Microsoft/vscode/issues/12076#issuecomment-340037747

Reply:
We recently implemented Sass linting to EUI and had this setting off (primarily because it allowed for multiple closing new lines). I'm not necessarily for or against it in any way, but if you feel strongly about it I'd raise it as a discuss issue in that Repo, since that linter will be soon making it's way to Kibana.

https://github.com/elastic/eui/blob/master/.sass-lint.yml#L68

Reply:
Thanks for the clarification, should have noted that was just a nitpick, not a blocker for the PR :)